contracts/contracts/validator/Mock_Uptime_feed.cairo

```diff
@@ -0,0 +1,53 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.math_cmp import is_le
+from starkware.starknet.common.syscalls import (
+    get_block_timestamp,
+)
+
+from ocr2.interfaces.IAggregator import Round
+from starkware.cairo.common.alloc import alloc
+
+@storage_var
+func l1_validator_() -> (address: felt):
+end
+
+@storage_var
+func round_() -> (round: Round):
+end
+
+@constructor
+func constructor{
+    syscall_ptr : felt*,
+    pedersen_ptr : HashBuiltin*,
+    range_check_ptr,
+}():
+    return ()
+end
+
+# receive and handle messages from L1
+@l1_handler
+func update_status{pedersen_ptr : HashBuiltin*, syscall_ptr : felt*, range_check_ptr}(from_address : felt, status : felt, timestamp : felt):
```

Comment:
1. What's the purpose of this `Mock_Uptime_feed.cairo` contract?
2. What does it enables us to test that we can't achieve with the `Mock_Aggregator.cairo`?

They should both implement the same interface `IAggregator.cairo`, one that should be used by users.